src/components/Settings/Setting.ts

```diff
@@ -0,0 +1,8 @@
+import { IconProps } from "assets/settings"
+
+export interface Setting {
+    title: string
+    subtitle?: string
+    icon?: IconProps
+    callback?: () => void
+}
```

Comment:
anche questo va spostato dai componenti, più ragionevolmente nel file in `utils` che ho descritto in un altro commento.

inoltre non so se `Setting` è il nome migliore, questo si riferisce specificatamente alla UI, nella mia testa `Settings` è la vera interfaccia dell'app, se non capita di usare la versione singola altrove immagino vada bene, però pensaci se magari troviamo un nome più sensato 🤷‍♂️

Reply:
Ok, SettingOptions o SettingProps può andare? siccome rappresenta i campi della UI corrispondenti a un setting

Reply:
si SettingOptions mi piace bravo